External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -226,7 +226,8 @@ function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, cha
 %% ====================================================================================
 %% 3. Create new NIfTI
 if ~isempty(changeMat)
-	newNifti.mat = changeMat;
+	newNifti.mat  = changeMat;

```

Comment:
mat0 should be the original orientation, as it comes from the scanner. This is useful for provenance checks in the orientation changes, e.g. for checking if there is a left-right flip, so theoretically we should never change mat0.

mat is whatever we want it to be, so can be changed

Reply:
Good point. We keep mat0 as it is... Except in this case. dcm2niix really messes up import and creates a completely incorrect mat and mat0 that have nothing to do with the scanner orientation. So we need to change both in this case. So I'm adding mat0 as an extra parameter to xASL_io_SaveNifti and comment that this is exceptional.

Reply:
Yes. Annoying, but dcm2niix has a lot of bugs and shortcomings that are not going to be fixed in the near future... So we have to accept that fact.